File: mario_more/mario.c
#include <cs50.h>
#include <stdio.h>
int height = 0;
int main(void)
{
    do
    {
        height = get_int("What's the height: ");
    }
    while (height < 1 || height > 8);
    // Making bricks
    for (int i = 0; i < height; i++)
    {
        // left spaces
        for (int j = 0; j < height - i - 1; j++)
        {
            printf(" ");
        }

        // left bricks
        for (int k = 0; k <= i; k++)
        {
            printf("#");
        }

        // between spaces
        printf("  ");

        // right bricks
        for (int l = 0; l <= i; l++)
        {
            printf("#");
        }
        // next line
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    height = get_int("What's the height? ")
    if 1 <= height <= 8:
        break
for i in range(height):
    for j in range(height - i - 1):
        print(" ", end="")
    for k in range(i + 1):
        print("#", end="")
    print("  ", end="")
    for l in range(i + 1):
        print("#", end="")
    print()
